report/countryList.js

```diff
@@ -0,0 +1,292 @@
+const countries = {
```

Comment:
there is already a file with mapping between country codes and country names https://github.com/konturio/geocint-mapaction/blob/main/scripts/ma_dictionaries.py
so I think we should rather move it to json and use it from both js and python scripts.

Reply:
I got this list from that python code.
Sure will discuss with Kirill to use json country codes also on python.

Reply:
used json file and fetch api for countrylist